File: Kwonsaebom/Week2/problem2.js
// https://www.acmicpc.net/problem/2751

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];

rl.on("line", (line) => {
  input.push(Number(line));

  if (input.length - 1 === input[0]) rl.close();
});

rl.on("close", () => {
  const count = input[0];
  const numbers = input.slice(1);

  numbers.sort((a, b) => a - b);

  console.log(numbers.join("\n"));
});

File: Hanseungwoo/Week2/problem2.py
# https://www.acmicpc.net/problem/2751

import sys  

n = int(sys.stdin.readline()) 
 
nums = []  

for _ in range(n):
    num = int(sys.stdin.readline())  
    nums.append(num)

nums.sort()  

for num in nums:
    print(num) 